common/utils/pulumi.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func PulumiDependsOn(resources ...pulumi.Resource) pulumi.ResourceOption {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is the advantage of this ? Is this to potentially hide `pulumi` to callers ? So importing `test-infra-definitions/utils` rather than `pulumi` ?

Reply:
Just because it's more practical than typing `[]pulumi.Array{}` every single time.